server_admin/topics.adoc

```diff
@@ -16,10 +16,10 @@ include::topics/authentication/kerberos.adoc[]
 include::topics/authentication/x509.adoc[]
 include::topics/authentication/webauthn.adoc[]
 include::topics/sso-protocols.adoc[]
-include::topics/sso-protocols/oidc.adoc[]
-include::topics/sso-protocols/saml.adoc[]
-include::topics/sso-protocols/saml-vs-oidc.adoc[]
-include::topics/sso-protocols/docker.adoc[]
+include::topics/clients/assembly-client-oidc.adoc[]
```

Comment:
Looks like this include is in the wrong place.  It goes in the file:  clients/assembly-managing-clients.adoc.